include/dlaf/gpu/blas/gpublas.h

```diff
@@ -0,0 +1,145 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2024, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+/// @file
+/// Provides gpublas wrappers for BLAS operations.
+
+#ifdef DLAF_WITH_GPU
+#include <cstddef>
+#include <utility>
+
+#include <whip.hpp>
+
+#include <dlaf/gpu/blas/api.h>
+#include <dlaf/gpu/blas/error.h>
+#include <dlaf/util_cublas.h>
+
+#ifdef DLAF_WITH_HIP
+
+#include <pika/async_cuda/detail/cuda_event_callback.hpp>
```

Comment:
This makes me sad, but it's our fault... At least there is a header included now. I think by now we have to accept that this is part of the public API. I've increased the priority of https://github.com/pika-org/pika/issues/887.